js/Utils.js

```diff
@@ -87,3 +87,22 @@ function ago(epoch) {
         }
     }
 }
+
+function netTypeToString(){
+    var nettype = appWindow.persistentSettings.nettype;
+    return nettype == NetworkType.TESTNET ? qsTr("Testnet") : nettype == NetworkType.STAGENET ? qsTr("Stagenet") : qsTr("Mainnet");
```

Comment:
`qrc:/js/Utils.js:94: ReferenceError: NetworkType is not defined`

Reply:
Fixed